test/resources/examples/users/src/main/java/org/mycompany/services/todo/config/SecurityConfiguration.java

```diff
@@ -0,0 +1,72 @@
+package org.mycompany.services.todo.config;
+
+import com.kapeta.spring.config.JWKInternalKeyStore;
+import com.kapeta.spring.config.JWKInternalKeyStoreProvider;
+import com.nimbusds.jose.JOSEException;
+import com.nimbusds.jose.jwk.JWKSet;
+import com.nimbusds.jose.jwk.KeyUse;
+import com.nimbusds.jose.jwk.gen.RSAKeyGenerator;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.text.ParseException;
+import java.util.UUID;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class SecurityConfiguration {
+
+    @Bean
+    public JWKInternalKeyStoreProvider jwkPublicKeySetProvider() {
+        return () ->
+            new JWKInternalKeyStore(
```

Comment:
Let's go with 
```
issuer: 'https://example.auth.kapeta.com',
audience: 'https://example.kapeta.com',
```

Fixing in node

Reply:
https://github.com/kapetacom/codegen-target-nodejs/pull/54

Reply:
Done.